master/networking/vxlan-ipip.md

```diff
@@ -40,6 +40,7 @@ You can configure each IP pool with different encapsulation configurations. Howe
 
 - [Configure IP in IP encapsulation for only cross subnet traffic](#configure-ip-in-ip-encapsulation-for-only-cross-subnet-traffic)
 - [Configure IP in IP encapsulation for all inter workload traffic](#configure-ip-in-ip-encapsulation-for-all-inter-workload-traffic)
+- [Configure VXLAN encapsulation for only cross subnet traffic](#configure-vxlan-encapsulation-for-only-cross-subnet-traffic)
```

Comment:
Line 35 in this file needs updating, but I can't add a comment to an unchanged line